rs-react-app/src/components/Header/Header.tsx

```diff
@@ -0,0 +1,61 @@
+import { Link } from 'react-router-dom';
+import styles from './Header.module.css';
+import useLocalStorage from '../../hooks/useLocalStorage';
+
+const Header = ({
+  onSearchSubmit,
+}: {
+  onSearchSubmit: (term: string) => void;
+}) => {
+  const [searchValue, setSearchValue] = useLocalStorage('searchValue', '');
+
+  const handleSearchSubmit = () => {
+    const trimmedValue = searchValue.trim().toLowerCase();
+    localStorage.setItem('searchValue', trimmedValue);
+    onSearchSubmit(trimmedValue);
+  };
+
+  const handleKeyDown = (e: React.KeyboardEvent<HTMLInputElement>) => {
+    if (e.key === 'Enter') {
+      handleSearchSubmit();
+    }
+  };
+
+  return (
+    <header className={styles.header}>
+      <nav className={styles.navContainer}>
+        <ul className={styles.navList}>
+          <li>
+            <Link to="/" className={styles.navLink}>
+              Home
+            </Link>
+          </li>
+          <li>
+            <Link to="/about" className={styles.navLink}>
+              About
+            </Link>
+          </li>
+        </ul>
+      </nav>
```

Comment:
Instead of hardcoding each <li>, consider storing the link data in an array and rendering it with .map(). This will make the navigation easier to maintain and extend.